src/frontend/magnify/src/components/profile/IdentityBlock/IdentityBlock.tsx

```diff
@@ -17,9 +16,9 @@ export default function IdentityBlock({ margin = { vertical: 'small' } }: Identi
   return (
     <Card background="white" margin={margin} pad={isSmall ? 'medium' : 'small'}>
       <Box align={'center'} direction={isSmall ? 'column' : 'row'} justify={'center'}>
-        <Box margin={{ vertical: 'auto', horizontal: 'medium' }}>
-          <AvatarForm key={user?.id || ''} id={user?.id} />
-        </Box>
+        {/*<Box margin={{ vertical: 'auto', horizontal: 'medium' }}>*/}
+        {/*  <AvatarForm key={user?.id || ''} id={user?.id} />*/}
+        {/*</Box>*/}
```

Comment:
Hmmm... I assume this code is removed temporarily but why not remove it completely then you will be able to find this commit from this commit later, WDYT ?

If you prefer to let this code commented, that's ok but add a comment to explain why this code is currently disabled.